### ISSUE TITLE
Feature request: Making accuracy limit

### ISSUE THREAD

**Issue by elmircode** [aradzie/keybr.com#250]
### Proposal for New Update: Accuracy Limit Feature

**Objective**: Improve typing accuracy by introducing a limit on the number of allowable errors.

**Feature Description**:
- **Accuracy Limit**: Users can make up to 3 errors during a typing session. If exceeded, they must restart the exercise.
- **Customizable Threshold**: Default limit set at 3 errors, adjustable in settings.
- **Real-time Error Count**: Display error count on the screen.

**User Interface Changes**:
- Add an error counter on the typing interface.
- Include a settings option for adjusting the accuracy limit.
- Display a restart prompt when the error limit is reached.

**Benefits**:
- Encourages users to focus on accuracy.
- Provides immediate feedback for improvement.
- Helps build a foundation for faster, error-free typing over time.

I wish u answer me and make a new feature

**Comment by WhaleKit** [comment]
Alternatively, I'd suggest a "target accuracy" similar to "target speed": if a key falls below it, it will become "current key" for practice until target accuracy is reached
disabling existing option "Stop Cursor On Error" helps by adding "natural penalty" of having to reach backspace, but
- it matters less on low WPM if user's hands learned to locate backspace quickly
- on layouts where it's not easily accessible (such as widely used is ANSI or ISO) it affects keys unevenly, since backspace is easier to reach it if your finger is already near it (e.g. by mistyping ']' on qwerty)

**Comment by orehzzz** [comment]
It would be nice to have an adjustable target accuracy, which could act the same as a target speed. If implemented, both speed and accuracy could be enabled/disabled independently, so you could use only one of them (or both)

**Comment by DRKolev-code** [comment]
This would be great as I am currently hitting ESC to accommodate this case.

**Comment by Lurler** [comment]
Yes, adding target accuracy requirement to options is really needed.

**Comment by j-standfast** [comment]
+1
